sphinx/themes/agogo/static/agogo.css.jinja

```diff
@@ -439,6 +439,19 @@ div.figure p.caption span.caption-text,
 figcaption span.caption-text {
 }
 
+#skiplink {
+  position: absolute;
+  top: -1000%;
+  left: -1000%;
+  padding: 1em;
+  background-color: #fff;
+}
+
+#skiplink:focus {
+  top: 10px;
+  left: 10px;
+}
```

Comment:
For `agogo` and a few other built-in themes it's necessary to duplicate the CSS because they don't inherit `basic.css`. When that stylesheet is inherited we don't need the duplication.